src/main/java/sim/coffee/Logger.java

```diff
@@ -35,6 +38,18 @@ public void add(Order[] o, OrderState state) {
 		case PROCESSED:
 			log.append(String.format("%-15s", "Processed"));
 			break;
+		
+		case ENTERKITCHEN:
+			log.append(String.format("%-15s", "Enter kitchen"));
```

Comment:
I feel like the way our logger is written is also very hardcoded and incohesive (consider: why should the logger be aware of these context specific things). Probably a good case for a strategy pattern (but I think we can go with this for now and tidy it up as a future PR).

Reply:
The impression I got from the coursework requirements was that they want to see the specific events that are taking place and my thinking was that Server/ Kitchen classes need a way to indicate what process is taking place hence the enums.

Reply:
I agree with your interpretation of the requirement, but I mean to say I'm not sure the logger should have these hardcoded events (very use-case specific and means every time we want to log something new the logger code needs a new case).

That said, I really think we should get this PR in and handle it afterwards.

Reply:
Ah, I see what you mean. Don’t see how it would work without the hardcoded events just now but I imagine you have an idea 😅